openapi/nexus.json

```diff
@@ -13706,22 +13706,37 @@
         ]
       },
       "Sled": {
-        "description": "Client view of a [`Sled`]",
+        "description": "An operator's view of a sled.",
         "type": "object",
         "properties": {
           "baseboard": {
             "$ref": "#/components/schemas/Baseboard"
           },
+          "cpus": {
+            "description": "The number of online processors available to the sled.",
+            "type": "integer",
+            "format": "int64"
```

Comment:
Why is the number of CPUs a signed value. What does a negative number of CPUs mean?